lib/location/Kconfig

```diff
@@ -28,14 +28,18 @@ config LOCATION_METHOD_WIFI
 	select NET_MGMT
 	select NET_MGMT_EVENT
 	select NET_MGMT_EVENT_INFO
-	select NET_L2_WIFI_MGMT
+	select NET_L2_WIFI_MGMT if !WIFI_NRF700X
```

Comment:
Even for WIFI_NRF700X we need this right?

Reply:
Seems that this is not needed anymore. It used to complain multiple definitions for net_mgmt but maybe something has changed since I put this condition here.
I will removed it.

Reply:
Yes, earlier for WIFI we used to use `NET_L2_NRF_WIFI_MGMT`, so, `NET_L2_WIFI_MGMT` had to be disabled, but this is now changed to use `NET_L2_WIFI_MGMT`.